lib/cassia/response_handlers/close_scan.rb

```diff
@@ -0,0 +1,36 @@
+module Cassia
+  module ResponseHandlers
+    class CloseScan
+      def initialize(access_controller, aps: )
+        @access_controller = access_controller
+        @aps = aps
+      end
+
+      def handle(response)
+        if response.success?
+          handle_success
+        else
+          handle_failure(response)
+        end
+        response.success?
+      end
+
+      private
+
+      def handle_success
+        if @aps == '*'
```

Comment:
maybe we should put this into a helper function since it's getting used throughout our code. You could create a new file called `router_helpers.rb` and put in a function that is called `get_routers` or something and looks like
```
def get_routers(routers, aps)
  return routers if aps == '*'
  routers.select { |router| aps.include?(router.mac) }
end
```

Then call this function in each of these response handlers to get the list of routers to update

We made a string helper https://github.com/rocketsofawesome/newgistics-ruby/blob/master/lib/newgistics/string_helper.rb in this gem for reusable string functions, similar premise as this`routers_helper.rb`